lib/utilities/get-config.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+let Yam = require('yam');
+let Project = require('../models/project');
+
+module.exports = function getConfig(configOverride) {
+  if (configOverride) {
+    return configOverride;
+  }
+  return new Yam('ember-cli', {
```

Comment:
I needed to bring the `new Yam()` into the `getConfig` function because it was failing the `tests/acceptance/pods-destroy-test.js` tests otherwise. Preferably the `new Yam` would happen at the module scope but in the pod-destroy tests the config wasn't being re-calculated after modifying the `.ember-cli` file.